scalding-core/src/main/scala/com/twitter/scalding/Job.scala

```diff
@@ -77,6 +77,7 @@ object Job {
  * these functions can be combined Monadically using algebird.monad.Reader.
  */
 class Job(val args: Args) extends FieldConversions with java.io.Serializable {
+  Tracing.init()
```

Comment:
The former, they should extend a trait. If they use driven they will know its there. Why have tracing code running for everyone as this would have? 

(Indeed it seems like tracing here should be a trait anyway that gets extended from. also can allow its methods easier overridden)